projects/dot-custom-elements/src/utils/utils.ts

```diff
@@ -1,4 +1,19 @@
-
 export function generateId(): number {
     return Date.now().valueOf();
 }
+
+export interface DotOption {
+    label: string;
+    value: string;
+}
+
+export function getItemsFromString(rawString: string): DotOption[] {
```

Comment:
This fail if we have spaces in the string of label and/or values:

<img width="518" alt="Screen Shot 2019-03-27 at 4 13 17 PM" src="https://user-images.githubusercontent.com/751424/55115914-49d77f00-50ab-11e9-92a4-e310aec58ee3.png">

Reply:
fixed, fyi the data that comes from the BE is in this format:
![image](https://user-images.githubusercontent.com/37185433/55121812-dc831880-50c1-11e9-89ad-d62260889b71.png)

Reply:
Yup... my chrome ext change that.